src/main/java/interface_adapter/search/SearchViewModel.java

```diff
@@ -2,8 +2,10 @@
 
 import interface_adapter.ViewModel;
 import interface_adapter.note.NoteState;
+
 import java.beans.PropertyChangeListener;
 import java.beans.PropertyChangeSupport;
+import java.util.List;
 
```

Comment:
```suggestion

import java.beans.PropertyChangeListener;
import java.beans.PropertyChangeSupport;
import java.util.List;

import interface_adapter.ViewModel;
import interface_adapter.note.NoteState;

```
rearrange order for checkstyle